src/register/RegistrationPage.jsx

```diff
@@ -443,173 +436,123 @@ class RegistrationPage extends React.Component {
-                  ariaInvalid={this.state.errors.country !== ''}
-                  invalidMessage={intl.formatMessage(messages['country.validation.message'])}
-                  className="mb-0 data-hj-suppress"
-                  value={this.state.country}
-                  onBlur={(e) => this.handleOnBlur(e)}
-                  onChange={(e) => this.handleOnChange(e)}
-                  selectOptions={this.getCountryOptions()}
-                  inputFieldStyle="border-gray-600"
+        <div className="mw-xs mt-3">
+          {this.renderErrors()}
+          {currentProvider && (
+            <ThirdPartyAuthAlert
+              currentProvider={currentProvider}
+              platformName={this.props.thirdPartyAuthContext.platformName}
+              referrer={REGISTER_PAGE}
+            />
+          )}
+          <Form>
+            <FormGroup
+              name="name"
+              value={this.state.name}
+              handleBlur={this.handleOnBlur}
+              handleChange={this.handleOnChange}
+              errorMessage={this.state.errors.name}
+              floatingLabel={intl.formatMessage(messages['registration.fullname.label'])}
+            />
+            <FormGroup
+              name="username"
+              value={this.state.username}
+              handleBlur={this.handleOnBlur}
+              handleChange={this.handleOnChange}
+              errorMessage={this.state.errors.username}
+              helpText={[intl.formatMessage(messages['help.text.username.1']), intl.formatMessage(messages['help.text.username.2'])]}
+              floatingLabel={intl.formatMessage(messages['registration.username.label'])}
+            />
+            <FormGroup
+              name="email"
+              value={this.state.email}
+              handleBlur={this.handleOnBlur}
+              handleChange={this.handleOnChange}
+              errorMessage={this.state.errors.email}
+              helpText={[intl.formatMessage(messages['help.text.email'])]}
+              floatingLabel={intl.formatMessage(messages['registration.email.label'])}
+            />
+            {!currentProvider && (
+              <PasswordField
+                name="password"
+                value={this.state.password}
+                handleBlur={this.handleOnBlur}
+                handleChange={this.handleOnChange}
+                errorMessage={this.state.errors.password}
+                floatingLabel={intl.formatMessage(messages['registration.password.label'])}
+              />
+            )}
+            <FormGroup
+              as="select"
+              name="country"
+              value={this.state.country}
+              handleBlur={this.handleOnBlur}
+              handleChange={this.handleOnChange}
+              errorMessage={this.state.errors.country}
+              floatingLabel={intl.formatMessage(messages['registration.country.label'])}
+              trailingElement=<Icon src={ExpandMore} />
```

Comment:
This can be removed, it's adding a double icon.

Reply:
Seems like it is only happening on safari, I will remove the default that appears on safari